prime-router/docs/standard-operating-procedures/adding-conformance-profile.md

```diff
@@ -0,0 +1,86 @@
+# Adding a New Conformance Profile in ReportStream
```

Comment:
Even with our pivot to erring on the side of less explicit support for conformance profiles, we still agreed there may one day be a situation where we need to add explicit support for a conformance profile, so I think this SOP is still relevant